package.json

```diff
@@ -163,10 +172,9 @@
     "simple-git": "^3.27.0",
     "stream-browserify": "^3.0.0",
     "style-loader": "^4.0.0",
-    "ts-jest": "^29.3.4",
-    "typescript": "^5.8.3",
-    "webpack": "^5.99.8",
-    "webpack-cli": "^6.0.1"
+    "typescript": "^5.7.3",
+    "vite": "^4.4.1",
+    "vitest": "^0.34.3"
```

Comment:
that is an ancient vitest version and it does not work with the vscode testing integration - updating it makes it work which is very nice